src/types/journal.ts

```diff
@@ -0,0 +1,51 @@
+import { AUTH_MODE, NODE_TYPES } from 'common/enums'
+
+export default /* GraphQL */ `
+  extend type Mutation {
+    putJournal(input: PutJournalInput!): Journal! @auth(mode: "${AUTH_MODE.oauth}") @rateLimit(limit:2, period:300)
```

Comment:
confirm with product or CC on rate limiting?

Reply:
just a temp value by now,  will  be update in future PR